docs/developer/file-structure.rst

```diff
@@ -28,58 +29,179 @@ The PyMedPhys repository is generally structured as follows:
+                   source code library. *changelog.md* determines the text
+                   presented on the `Release Notes`_ documentation page.
+
+:``setup.py``: A Python script that facilitates easy installation of the
+               PyMedPhys library as a package for users and contributors alike.
+
+You'll quickly note from a cursory look through PyMedPhys that there are
+actually many more top-level files. Most of these help configure specific
+tasks, such as automated testing. They are probably less critical and have been
+omitted from this page in the interest of brevity.
+
+(Most of) the rest of PyMedPhys is arranged in the following directories:
+
+:``docs/``: Contains most of PyMedPhys' documentation. The files within *docs/*
+            make up the text you are reading right now! A documentation
+            generator called Sphinx converts the files in *docs/* into
+            human-readable format. The same tool also pulls the docstrings from
+            PyMedPhys' source code and displays them in the documentation
+            pages.
+
+:``notebooks/``: Contains a series of experimental `Jupyter notebooks`_.
+                 Jupyter notebooks provide a highly convenient way to
+                 experiment with code. Some PyMedPhys contributors prefer to
+                 code in Jupyter notebooks to zero in on a solution before
+                 attempting to add code to the PyMedPhys library. *notebooks/*
+                 is a convenient place to store these experimental notebooks,
+                 permitting both remote and collaborative raw development. 
+
+:``scripts/``: Contains (or will contain) various scripts ready-made to run and
+               perform simple tasks.
+
+:``src/pympedhys/``: Contains the PyMedPhys source code library or *package*.
+                     Code within this constitutes "PyMedPhys proper". In theory
+                     (though not yet in practice), this code has been tested
+                     and documented. Changes to code in this folder are tracked
+                     in ``changelog.md``. See `The PyMedPhys Source Code
+                     Package`_ section below for further details, especially on
+                     structure.
+
+:``tests/``: Contains the PyMedPhys suite of automated tests. Any code present
+             in *src/pymedphys/* should be covered by tests in this directory.
+             Automated testing is essential for effective `continuous
+             integration`_, which is a core development philosophy of
+             PyMedPhys. If you would like to make meaningful contributions to
+             PyMedPhys - and become a much better developer as a result - it
+             pays to get very familiar with automated testing and the code
+             within this directory.
+
+
+.. _`the PyMedPhys GitHub page`: https://github.com/pymedphys/pymedphys
+.. _`Release Notes`: /getting-started/changelog.html
```

Comment:
Need relative links here ../getting-started etc ...

Reply:
Interesting... those links worked for me when building locally?

Reply:
Yup, that's because readthedocs adds "latest" and "en" to the root. Locally you're just using the root path directly.